.github/workflows/e2e-tests.yaml

```
@@ -52,11 +61,27 @@ jobs:
             CSI_IMAGE_REPOSITORY=ghcr.io/${{ github.repository }} \
             ADDITIONAL_ARGS="--junit-report=./test-results/e2e-tests-results.xml"
 
+      - name: Copy S3 logs to artifacts directory
+        if: always()
+        run: |
+          mkdir -p artifacts/logs/s3
+          cp -r .github/scality-storage-deployment/logs/s3/* artifacts/logs/s3/ 2>/dev/null || true
+
+      - name: Upload S3 logs to Scality artifacts
+        if: always()
+        uses: scality/action-artifacts@v4
+        with:
+          method: upload
+          url: https://artifacts.scality.net
+          user: ${{ secrets.ARTIFACTS_USER }}
+          password: ${{ secrets.ARTIFACTS_PASSWORD }}
+          source: artifacts
+
       - name: Upload test results to Codecov
         if: ${{ always() }}
         uses: codecov/test-results-action@v1
         with:
           token: ${{ secrets.CODECOV_TOKEN }}
           file: ./tests/e2e/test-results/e2e-tests-results.xml
-          flags: e2e_tests
+          flags: e2e_tests,cloudserver_${{ matrix.github_varirable_name }}
```

This will render as `cloudserver_CLOUDSERVER_RING_9_4`, would go either:

`cloudserver_9.4` with:
```suggestion
          flags: e2e_tests,cloudserver_${{ matrix.ring_version }}
```

Or `CLOUDSERVER_RING_9_4` with:

```suggestion
          flags: e2e_tests,${{ matrix.github_varirable_name }}
```

---

Changed in https://github.com/scality/mountpoint-s3-csi-driver/pull/61/commits/dc0b08b5d2441beb41a77fd1a4ebdaf2b94d570e
`matrix.ring_version`